apps/bl-ap-imgproc.bash

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+export PYTHONUNBUFFERED=yes
+
+# CPU usage estimate (ps aux):
+
+/usr/local/bin/sirius-ioc-bl-ap-imgproc.py --devname CAX:B:BASLER01 \
+| tee /ioc-logs/sirius-ioc-bl-ap-imgproc.log &
```

Comment:
I think the logfile could be: sirius-ioc-cax-ap-basler1.log or sirius-ioc-bl-ap-imgproc-cax-basler1.log, just to identify the device, in case we have more instances of this IOC

Reply:
I added the script for DVF1 at CAX at machine-applications and changef the log filenames here as well.